sync-core/src/main/java/com/cloudant/sync/datastore/AttachmentNotSavedException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2015 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.sync.datastore;
+
+/**
+ * Created by Rhys Short on 05/02/15.
+ */
+public class AttachmentNotSavedException extends AttachmentException {
+
+    public AttachmentNotSavedException(){}
+
+    public AttachmentNotSavedException(String message){
+        super(message);
+    }
+
+    public AttachmentNotSavedException(Exception causedBy){
+        super(causedBy);
+    }
+
+    public AttachmentNotSavedException(String message, Exception casuedBy){
+        super(message,casuedBy);
+    }
```

Comment:
Looking at the code usages, its not really worth adding helper constructors, it looks like they will only be called once